docs/source/getting_started/platform.rst

```diff
@@ -194,13 +194,47 @@ need to look like this:
         ]
     }
 
+If you don't have your data in coco format yet, but e.g. as a pandas dataframe,
+you can use a simple script to translate it to the coco format:
```

Comment:
Maybe add an introductory sentence that tells you that we use coco format for metadata. Otherwise it is pretty confusing if you just start with coco format in a tutorial on metadata. A link to some coco format resource would be nice as well.

Reply:
Look at the docs before this change, it is included there.